sw/airborne/boards/px4fmu/chibios/v5.0/board.h

```diff
@@ -57,7 +57,7 @@
 #endif
 
 /* Default powerbrick values */
-#define DefaultVoltageOfAdc(adc) ((3.3f/4096.0f) * 10.3208191126f * adc)
+#define DefaultVoltageOfAdc(adc) ((3.3f/4096.0f) * 18.0f * adc)
```

Comment:
It was not wrong before. A different value should IMHO have been set in airframe file like:

```
<section name="BAT">
...
    <!-- Changed since this airframe has a non default 3DR type powerbrick for Current and Voltage measurement -->
    <define name="VoltageOfAdc(_adc)" value="(3.3f/4096.0f) * 18.0f * adc"/>
...
```